tests/Mono.Android-Tests/Runtime-AppBundle/Mono.Android-TestsAppBundle.csproj

```diff
@@ -65,76 +65,88 @@
     </AndroidResource>
-    <AndroidResource Include="$(_MonoAndroidTest)Resources\layout\lowercase_custom.axml">
+    <AndroidResource Include="..\Resources\layout\lowercase_custom.axml">
       <Link>Resources\layout\lowercase_custom.axml</Link>
     </AndroidResource>
-    <AndroidResource Include="$(_MonoAndroidTest)Resources\xml\XmlReaderResourceParser.xml">
+    <AndroidResource Include="..\Resources\xml\XmlReaderResourceParser.xml">
       <Link>Resources\xml\XmlReaderResourceParser.xml</Link>
     </AndroidResource>
-    <AndroidResource Include="$(_MonoAndroidTest)Resources\layout\FragmentFixup.axml">
+    <AndroidResource Include="..\Resources\layout\FragmentFixup.axml">
       <Link>Resources\layout\FragmentFixup.axml</Link>
     </AndroidResource>
-    <AndroidBoundLayout Include="$(_MonoAndroidTest)Resources\layout\Main.axml">
+    <AndroidBoundLayout Include="..\Resources\layout\Main.axml">
       <Link>Resources\layout\Main.axml</Link>
     </AndroidBoundLayout>
   </ItemGroup>
   <ItemGroup>
-    <AndroidAsset Include="$(_MonoAndroidTest)Assets\asset1.txt">
+    <AndroidAsset Include="..\Assets\asset1.txt">
       <Link>Assets\asset1.txt</Link>
     </AndroidAsset>
-    <AndroidAsset Include="$(_MonoAndroidTest)Assets\subfolder\asset2.txt">
+    <AndroidAsset Include="..\Assets\subfolder\asset2.txt">
       <Link>Assets\subfolder\asset2.txt</Link>
     </AndroidAsset>
-    <AndroidAsset Include="$(_MonoAndroidTest)Assets\subfolder\accept_request.png">
+    <AndroidAsset Include="..\Assets\subfolder\accept_request.png">
       <Link>Assets\subfolder\accept_request.png</Link>
     </AndroidAsset>
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-  <Import Project="Mono.Android-TestsAppBundle.targets" />
   <ItemGroup>
-    <ProjectReference Include="..\TestRunner.Core\TestRunner.Core.csproj">
+    <ProjectReference Include="..\..\TestRunner.Core\TestRunner.Core.csproj">
       <Project>{3cc4e384-4985-4d93-a34c-73f69a379fa7}</Project>
       <Name>TestRunner.Core</Name>
     </ProjectReference>
-    <ProjectReference Include="..\TestRunner.NUnit\TestRunner.NUnit.csproj">
+    <ProjectReference Include="..\..\TestRunner.NUnit\TestRunner.NUnit.csproj">
       <Project>{CB2335CB-0050-4020-8A05-E9614EDAA05E}</Project>
       <Name>TestRunner.NUnit</Name>
     </ProjectReference>
-    <ProjectReference Include="$(_MonoAndroidTest)Java.Interop-Tests\Java.Interop-Tests.csproj">
+    <ProjectReference Include="..\Java.Interop-Tests\Java.Interop-Tests.csproj">
       <Project>{6CB00820-A66B-43E5-8785-ED456C6E9F39}</Project>
       <Name>Java.Interop-Tests</Name>
     </ProjectReference>
-    <ProjectReference Include="$(_MonoAndroidTest)Mono.Android-Test.Library\Mono.Android-Test.Library.csproj">
+    <ProjectReference Include="..\Mono.Android-Test.Library\Mono.Android-Test.Library.csproj">
       <Project>{8CB5FF58-FF95-43B9-9064-9ACE9525866F}</Project>
```

Comment:
Is there a few more things that could move into `Mono.Android-Test.Shared.projitems` or `Directory.Build.targets` so we have less duplication?

I'm thinking the `@(ProjectReference)`, `@(AndroidResource)`, `@(AndroidAssets)`, and `@(AndroidBoundLayout)` items would all be nearly the same.

Reply:
Great idea, I moved these over to `Mono.Android-Test.Shared.projitems`. I also removed the duplicate resources that belonged to `Runtime-MultiDex` and all three projects now use the one shared set of resources and assets.